stories/components/alert/Alert.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as stories from './Alert.stories.js';
+
+<Meta of={stories} />
+
+# Alerts
+Use a notification banner to tell the user about something they need to know about, 
+but that’s not directly related to the page content.
+
+## When to use
+
+Use alerts to inform users of updates or changes to system status, or to acknowledge or 
+confirm user actions. While alerts are an effective method of communicating with users, 
+they are disruptive and should be used sparingly.
+
+## Best practices
+
+- Use the `role='alert'` attribute to support screen readers.
+- Alerts should not block a user from performing any other task on the screen.
+- Use the base `alert` class along with a color class that sets the background
+  color of the alert.
+- Do not use a notification banner to tell the user about validation errors - 
+  use an error message and error summary instead
+
+## Related components
+
+- Tooltip
+
+## Known issues
+
+`alert--orange` does not meet WCAG requirements for color contrast.
+
+## Basic implementation
+Use javascript to show the alert or hide it when a user clicks on the close button.
+
+<Story of={stories.basic} />
+
+## Colors
+Alerts come in a number of different colors. Use the following classes:
```

Comment:
I'd vote to only include the colors that would make an alert stand out visually and draw attention to itself, since we want these to be really visible to sighted users (and are announcing them for screen reader users). I would say only blue, black, and orange.